portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/remotewebapps/RemoteApps.testcase

```diff
@@ -517,6 +512,48 @@ definition {
 		}
 	}
 
+	@description = "LPS-159013 ES module custom element portlets render incorrect <script> node when dragged in the UI"
+	@priority = "4"
+	test CustomElementRendersCorrectlyWhenPlacedOnWidgetPage {
+		property portal.acceptance = "true";
+		property testray.component.names = "Frontend Dataset,Remote Apps";
+
+		task ("Given a custom element with "Use ES Modules" option created") {
+			JSONRemoteApp.addCustomElementRemoteAppEntry(
+				customElementCssurl = "https://liferay.github.io/liferay-frontend-projects/index.css",
+				customElementHtmlName = "vanilla-counter",
+				customElementName = "Vanilla Counter",
+				customElementProperties = "test-data-user=QAuser",
+				customElementURL = "https://liferay.github.io/liferay-frontend-projects/vanilla-counter/index.js",
+				customElementUseESM = "true");
+		}
+
+		task ("And Given a widget page created") {
+			JSONLayout.addPublicLayout(
+				groupName = "Guest",
+				layoutName = "My Widget Page");
+		}
+
+		task ("When adding the app portlet to the page") {
+			Navigator.gotoPage(pageName = "My Widget Page");
+
+			Click(locator1 = "ControlMenu#ADD");
+
+			Type(
+				locator1 = "NavBar#APPLICATION_SEARCH_FIELD",
+				value1 = "Vanilla Counter");
+
+			Portlet._clickAddPortlet(portletName = "Vanilla Counter");
```

Comment:
Had a bit of confusion here since in the title `when dragged in the UI` sounds like we'd want the test to drag and drop the widget. But this click follows the steps to reproduce in description so ok.

Reply:
that was the first idea, but after consulting with Iván, the error was appearing not only when dragging and dropping, but also when adding the portlet with the "+" add sign